vnext/Microsoft.ReactNative/Modules/I18nManagerModule.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "I18nManagerModule.h"
+#include <IReactDispatcher.h>
+#include <winrt/Windows.ApplicationModel.Resources.Core.h>
+#include <winrt/Windows.Globalization.h>
+#include "Unicode.h"
+
+namespace Microsoft::ReactNative {
+
+static const React::ReactPropertyId<bool> &SystemIsRTLPropertyId() noexcept {
+  static const React::ReactPropertyId<bool> prop{L"I18n", L"SystemIsRTL"};
```

Comment:
can all these be constexpr?

Reply:
No, we cannot use the constrexpr - internally the PropertyId constructor calls ReactPropertyBagHelper::GetName ABI API to create an atomized name that we use to index property value using a single pointer. We are spending some time for creating each PropertyId, but then the use of PropertyBag is more efficient.

---
In reply to: [429526933](https://github.com/microsoft/react-native-windows/pull/4999#discussion_r429526933) [](ancestors = 429526933)